homeassistant/components/verisure/lock.py

```diff
@@ -80,7 +80,7 @@ def update(self):
             "$.doorLockStatusList[?(@.deviceLabel=='%s')].lockedState",
             self._device_label)
         if status == 'UNLOCKED':
-            self._state = None
+            self._state = STATE_UNLOCKED
```

Comment:
We should instead remove the `state` property in the verisure lock entity class, and implement `is_locked`. Lock platforms should not overwrite the `state` property.